tactics/class_tactics.mli

```diff
@@ -34,23 +34,3 @@ val not_evar : constr -> unit Proofview.tactic
 val is_ground : constr -> unit Proofview.tactic
 
 val autoapply : constr -> Hints.hint_db_name -> unit Proofview.tactic
-
-module Search : sig
-  val eauto_tac :
```

Comment:
Why remove this? It's gonna be needed to plug eauto on it at some point

Reply:
Indeed, I could keep instead but I'll have to change the signature anyways.

Reply:
I think it should not be removed now. For example Equations only uses `typeclasses_eauto` in the same module but I've use cases where I could use this richer interface. And it'll be better to track the changes to the interface you'll make :)

Reply:
OK :+1: